app/build.gradle

```diff
@@ -1,17 +1,17 @@
 import groovy.json.JsonSlurper
 
 buildscript {
-  repositories {
```

Comment:
Please fix indentation here and in many other places. 2 space indentation, 4 space line continuation.